exampleSite/config/_default/languages.yaml

```diff
@@ -3,9 +3,9 @@ en:
   weight: 1
   ### copyright (markdown can be used) ###
   copyright: "Copyright &copy; {{< current_year >}} Company Name. All Rights Reserved. [Dot Org Theme](https://github.com/cncf/dot-org-hugo-theme) for Hugo by [CNCF](https://www.cncf.io)."
+  title: "Site Title"
```

Comment:
Isn't this redundant considering it is already defined in `/config/_default/hugo.yaml`? Perhaps it should be removed from `hugo.yaml` if the intention is that one specifies the title per language.

Reply:
Fair point. Right now it's not clear which one takes precedence as well. @thetwopct any thoughts on removing it from `hugo.yaml`?

Reply:
From my tests the one in `/config/_default/hugo.yaml` takes precedence. I don't see the other on any page.

Reply:
From my tests it seems that the one in `languages.yaml` takes precedence, which makes sense to me. I'm fine with keeping it also in `hugo.yaml` in case there's a site that doesn't need a `languages.yaml` file.